functions/src/functions/enrollUser.test.ts

```diff
@@ -97,6 +97,11 @@ describeWithEmulators('function: enrollUser', (env) => {
       { uid: authUser.uid },
     )
 
+    const userService = env.factory.user()
+    const dbUser = await userService.getUser(authUser.uid)
+    expect(dbUser).to.exist
+    if (dbUser !== undefined) await userService.finishUserEnrollment(dbUser)
```

Comment:
We may want to test them separately in the future - for now, it is only tested to work when both actually run.

Reply:
#170